cmd/mavenBuild.go

```diff
@@ -255,7 +254,7 @@ func createBuildArtifactsMetadata(config *mavenBuildOptions, commonPipelineEnvir
 			} else {
 				coordinate.BuildPath = filepath.Dir(match)
 				coordinate.URL = config.AltDeploymentRepositoryURL
-				coordinate.PURL = getPurlForThePom(match)
+				coordinate.PURL = piperutils.GetPurl(match, mvnSimpleBomFilename+".xml")
```

Comment:
The bom path is wrong. For maven the bom will be inside target folder


bomPath := filepath.Join(filepath.Dir(match) + "/target/" + mvnSimpleBomFilename + ".xml")
coordinate.PURL = piperutils.GetPurl(match, mvnSimpleBomFilename+".xml")

Reply:
[201a9fa](https://github.com/SAP/jenkins-library/pull/5166/commits/201a9fa0be294c21389a38ef29de30ac43b2ce2a)

Reply:
Yep, I think that was the issue with the failing tests